### DISCUSSION TITLE
Is it intended that Tooltips close on key press of anything?

### DISCUSSION THREAD

**Discussion by dougmacknz** [adobe/react-spectrum#6866] [General]
https://react-spectrum.adobe.com/react-aria/Tooltip.html

If you tab to one of the buttons that has a tooltip, then press any key (e.g. shift), the tooltip closes.

Is this a bug or intended?

Discovered because I wanted to take a screenshot of a tooltip open and that involved a keyboard shortcut, which I was able to get around by opening the tooltip with mouse focus instead.

**Comment by snowystinger** [comment]
Looks like it was semi-intentional added in this PR https://github.com/adobe/react-spectrum/pull/5526

@devongovett as the author of that PR, do you recall anything more?

Otherwise, I think it makes sense. Any interaction should close the Tooltip in my opinion, and any key press could theoretically trigger some other action, including opening a Dialog.
